src/app/account/repo/checks/[id]/form.js

```diff
@@ -19,6 +21,21 @@ export default function Form({ id }) {
 }
 
 export function FormBadge({ src }) {
+  const [copy, setCoppy] = useState(false);
+  const copyHandle = () => {
+    const url = `![HealthCheck](${src})`;
+
+    // Use the Clipboard API to copy the URL
+    navigator.clipboard
+      .writeText(url)
+      .then(() => {
+        console.log("Copied to clipboard:", url);
+        setCoppy(true);
+      })
+      .catch((err) => {
+        console.error("Failed to copy: ", err);
```

Comment:
I don't think we need the console logs, but can display an error message (simialr to the `Copied!` but in red